data/IntroText.h

```diff
@@ -1,5 +1,5 @@
 #pragma once
-const char *const introTextData =
+const ByteString introTextData =
```

Comment:
Probably better as `String`.

Reply:
Probably correct but where it was used it was converted to a ByteString so didn't wan't to break anything, this was one of the first changes i made, can take a look into where it as used to possibly change to String